services/notification/types/commitstatus.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	"agola.io/agola/internal/sqlg"
+	"agola.io/agola/internal/sqlg/sql"
+)
+
+type Status string
```

Comment:
Status is too generic. Like already done in the gitsource api it could be called CommitState and rename the related fields to State instead of Status (both here and in the db objects).